live-build/config/hooks/vm-artifacts/90-raw-disk-image.binary

```diff
@@ -137,6 +137,7 @@ zfs create \
 	"rpool/ROOT/$FSNAME/root"
 
 zfs mount "rpool/ROOT/$FSNAME/root"
+mount --make-slave "$DIRECTORY"
```

Comment:
Since the placement of this call is critical, can you perhaps add a comment to explain why it _needs_ to be called here? (e.g. in contrast to calling it later like before this change).

I'd be OK more-or-less copying the PR description into the form of a comment here, if needed.